docker-compose.yml

```diff
@@ -1,54 +1,71 @@
 version: '3'
```

Comment:
"version" is obsolete. 

https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-obsolete